neural/run_tf.py

```diff
@@ -0,0 +1,43 @@
+from dataset import load
+from sklearn import cross_validation
+import argparse
+import tensorflow as tf
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-t", "--train", action="store_true", help="Train model with dataset otherwise load it.")
+args = parser.parse_args()
+# Load our dataset
+X, y = load()
+
+# Split dataset into train / test
+X_train, X_test, y_train, y_test = cross_validation.train_test_split(X, y, test_size=0.25)
+
+# Initialize the variables (i.e. assign their default value)
+init = tf.global_variables_initializer()
+
+# Build fully connected DNN
+with tf.Session() as session:
+    # Run the initializer
+    session.run(init)
+    with tf.device('/gpu:0'):
+        # Enable logging
+        tf.logging.set_verbosity(tf.logging.INFO)
+        # Create our classifier
+        feature_columns = [tf.contrib.layers.real_valued_column("", dimension=1024)]
+        classifier = tf.contrib.learn.DNNClassifier(feature_columns=feature_columns, hidden_units=[400, 400, 400, 400, 400, 400], n_classes=4, model_dir="model")
```

Comment:
Have you tried with fewer hidden units? I believe that for a task as simple as this, 1 or no hidden layer at all could be enough.

Reply:
Depending on how fast it runs, I'd suggest you to do a benchmarking on this and see what happens